specification/billing/resource-manager/Microsoft.Billing/privatepreview/2020-11-01-privatepreview/examples/BillingActivityList.json

```diff
@@ -0,0 +1,43 @@
+{
+  "parameters": {
+    "api-version": "2020-11-01-privatepreview",
+    "billingAccountName": "{billingAccountName}",
+    "startDate": "{startDate}",
+    "endDate": "{endDate}",
+    "activityLogEventCategory": "{activityLogEventCategory}"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/providers/Microsoft.Billing/billingAccounts/7303440/billingActivityList",
+            "name": "7303440",
+            "properties": {
+              "activityEventId": "ManageAccountAdd",
+              "billingAccountName": "7303440",
+              "createdOn": "2020-01-10 19:44:00.5300000",
+              "createdByPrincipalUserEmailAddress": "demouser@outlook.com",
+              "friendlyEventCategoryName": "Add Account",
+              "extendedProperties": "{\"AccountEmail\":\"demouser@outlook.com\",\"AccountAuthType\":\"OrganizationalAccount\",\"AccountName\":\"demouseraccount@miccrosoft.com\",\"AccountCostCenter\":\"ACM\"}"
```

Comment:
as per https://armwiki.azurewebsites.net/api_contracts/guidelines/openapi.html#oapi016-string-types-that-should-be-objects please model this as an object rather than a string